src/web-auth/index.js

```diff
@@ -283,6 +283,14 @@ WebAuth.prototype.validateAuthenticationResponse = function(options, parsedHash,
     if (validationError.error !== 'invalid_token') {
       return callback(validationError);
     }
+    if (!parsedHash.access_token) {
```

Comment:
how can you know with just this `!parsedHash.access_token` that the token is HS256 signed?

Reply:
ops, moved it below the .alg check